### PR TITLE
NO-TICKET: Add CORS header

### PR DESCRIPTION
**Pull request by pgurusinga**
Widget responds the Access-Control-Allow-Origin header

### CODE REVIEW

File: src/main/kotlin/de/bund/digitalservice/useid/config/SecurityHeadersFilter.kt
Comment (pgurusinga) [inline src/main/kotlin/de/bund/digitalservice/useid/config/SecurityHeadersFilter.kt:38]: Please refer to MDN:
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#the_http_response_headers 

> If the server specifies a single origin (that may dynamically change based on the requesting origin as part of an allowlist) rather than the "*" wildcard, then the server should also include Origin in the [Vary](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Vary) response header to indicate to clients that server responses will differ based on the value of the [Origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) request header.

**Review by mpanne — APPROVED** [review]
LGTM

**Comment by pgurusinga** [comment]
@mpanne could you please approve once again? you approved it when it was still in draft and I cannot merge it 😄

**Review by mpanne — APPROVED** [review]

**Pull request merged**
